src/diffusers/loaders/peft.py

```diff
@@ -296,11 +325,47 @@ def load_lora_adapter(self, pretrained_model_name_or_path_or_dict, prefix="trans
             if is_peft_version(">=", "0.13.1"):
                 peft_kwargs["low_cpu_mem_usage"] = low_cpu_mem_usage
 
+            if hotswap:
+                try:
+                    from peft.utils.hotswap import check_hotswap_configs_compatible, hotswap_adapter_from_state_dict
+                except ImportError as exc:
+                    msg = (
+                        "Hotswapping requires PEFT > v0.14. Please upgrade PEFT to a higher version or install it "
+                        "from source."
+                    )
+                    raise ImportError(msg) from exc
```

Comment:
Can we check the version instead of relying on exception? We have `is_peft_version`

https://github.com/huggingface/diffusers/blob/97abdd2210a540c2e71aee63c80a22723031cd57/src/diffusers/utils/import_utils.py#L808

Reply:
I did this on purpose, as it allows to test the feature by installing PEFT from main. Otherwise, we'd have to wait for the next PEFT release. Normally, I'd also avoid `try import ...` for the side effect, but at this point, PEFT is already imported, so that's not a factor.

If you still want me to change this, LMK.

Reply:
Can't we do `is_peft_version(">", "0.14.0")`?

Reply:
Ah yes, that should work, I'll fix.